BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Loop_Parts/Part_B/Part_B_PUEO.sh

```diff
@@ -248,4 +244,4 @@ else
 fi
 
 echo "Submitting XF jobs with batch size $batch_size"
-sbatch --array=1-${XFCOUNT}%${batch_size} --export=ALL,WorkingDir=$WorkingDir,RunName=$RunName,XmacrosDir=$XmacrosDir,XFProj=$XFProj,NPOP=$NPOP,indiv=$individual_number,indiv_dir=$indiv_dir,gen=${gen},SYMMETRY=$SYMMETRY,PSIMDIR=$PSIMDIR,batch_size=$batch_size,SingleBatch=$SingleBatch --job-name=${RunName} --time=${job_time} $job_file 
```

Comment:
What does cutting "indiv_dir=$indiv_dir" do? Is it already accounted for in the job script without passing this variable, or do we not need it anymore?

Reply:
Yeah, it isn't even defined in Part B, so it's just been passing an empty variable. The job script is where the variable actually gets assigned.